src/test/groovy/uk/gov/hmcts/dm/it/CreateDocumentIT.groovy

```diff
@@ -344,6 +349,137 @@ class CreateDocumentIT extends BaseIT {
             .get(url)
     }
 
+    @Test
+    void "CD13 (R1) As authenticated when i upload a text file or Tiff I get an icon in return"() {
```

Comment:
not sure about  this test why upload two document in one go?

it would be interesting to see if with say two different jpeg see what the endpoint is

Reply:
The reason to do two tests check in one test was as they were both getting the same icon.

Reply:
ok kool